code-challenges/401challenges/src/main/java/challenges/linkedlist/LinkedList.java

```diff
@@ -69,4 +69,82 @@ public ArrayList print(){
 
         return nodeValues;
     }
+
+    public void append(int data){
+        try
+        {
```

Comment:
Opening `{` should be on the same line as the thing they are opening:

`try {`